### PR TITLE
solution to xml error

### PR DESCRIPTION
**Pull request by mat10tng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,9 @@
   $ sudo apt-get install python3-pip
 4. install flask for python3
   $ sudo pip3 install flask
+Error
```
Comment (TabTabTab) [inline README.md]: Please create a separate section for Errors last in the document, such as ##Errors. And make sure to separate it with one newline from the above section (below the line "$ ./run.py").

File: README.md [hunk 2]
```diff
@@ -10,6 +10,9 @@
...
+1. lxml
```
Comment (TabTabTab) [inline README.md]: Please also follow the format of "\t$ "command"" when writing commands.

File: README.md [hunk 3]
```diff
@@ -10,6 +10,9 @@
   $ sudo apt-get install python3-pip
 4. install flask for python3
   $ sudo pip3 install flask
+Error
+1. lxml 
+   solution: pip3 install --upgrade lxml
```
Comment (TabTabTab) [inline README.md]: Please follow the format of "\t$ "command"" when writing commands.

File: README.md (hunk 2 above)
Comment (TabTabTab) [inline README.md]: "Error: ImportError: No module named 'lxml'"
Would be a more descriptive title

File: README.md (hunk 1 above)
Comment (mat10tng) [inline README.md]: K

File: README.md (hunk 3 above)
Comment (mat10tng) [inline README.md]: K

File: README.md (hunk 2 above)
Comment (mat10tng) [inline README.md]: K

File: README.md [hunk 4]
```diff
@@ -10,8 +10,9 @@
   $ sudo apt-get install python3-pip
 4. install flask for python3
   $ sudo pip3 install flask
-Error
-1. lxml 
+
+## Error
+1. lxml
```
Comment (TabTabTab) [inline README.md]: Please see comment on previous submission for guidelines

File: README.md [hunk 5]
```diff
@@ -12,8 +12,8 @@
   $ sudo pip3 install flask
 
 ## Error
-1. lxml
-   solution: pip3 install --upgrade lxml
+1. ImportError: No module named 'lxml'.
```
Comment (TabTabTab) [inline README.md:15]: Sweet! Merging! :+1:

**Pull request merged**